src/plugins/intel_gpu/src/graph/graph_optimizer/reorder_inputs.cpp

```diff
@@ -740,6 +740,41 @@ void reorder_inputs::run(program& p, reorder_factory& rf) {
         }
     };
 
+    const auto reorder_eltwise = [&p](typed_program_node<eltwise>& eltwise_node) {
```

Comment:
random spot)
* how does it work for static-shape models? Did you have a chance to review that? Does it happen only for dynamic shape models?
* Did you review to implement it from ngraph transformation?
* what about supporting such broadcast directly from eltwise? we already have quite complicated broadcasting from eltwise.
* What about inserting explicit reshape in case of numpy broadcast?

Reply:
- It can reproduce with static shape. It will be same as long as Eltwise has an NUMPY broadcasting.
  The 'reorder_inputs' pass need to add same reorder to align tensor size.  
- It checks preferred format. I think reorder_inpus pass is better.
- Sounds good with limited condition. I will try it.
- I think just reorder looks fine. Have we had logic to add reshape+reorder?